src/main/java/org/jitsi/videobridge/EndpointConnectionStatus.java

```diff
@@ -210,41 +210,40 @@ private void doMonitor()
             for (Videobridge videobridge : jvbs)
             {
                 Conference[] conferences = videobridge.getConferences();
-                for (Conference conference : conferences)
-                {
-                    List<Endpoint> endpoints = conference.getEndpoints();
-                    for (Endpoint endpoint : endpoints)
-                    {
-                        monitorEndpointActivity(endpoint);
-                    }
-                }
+                Arrays.stream(conferences)
+                    .forEachOrdered(
+                        conference ->
+                            conference.getEndpoints()
+                                .forEach(this::monitorEndpointActivity));
 
                 cleanupExpiredEndpointsStatus();
             }
         }
     }
 
-    private void monitorEndpointActivity(Endpoint endpoint)
+    private void monitorEndpointActivity(EndpointBase endpointBase)
     {
+        if (!(endpointBase instanceof Endpoint))
```

Comment:
this makes me wonder if it's worth renaming `Endpoint` to something that conveys a directly-connected endpoint vs a remote one?

Reply:
I wanted to avoid renaming Endpoint because it is used in so many places (including in property names)